test/common/multitenancy_load.go

```diff
@@ -0,0 +1,232 @@
+package common
+
+import (
+	goctx "context"
+	"fmt"
+	rhmiv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/test/resources"
+	testResources "github.com/integr8ly/integreatly-operator/test/resources"
+	routev1 "github.com/openshift/api/route/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"net/http"
+	"os"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const (
+	pollingTime = time.Second * 5
+	realmName   = "rhd"
+)
+
+var multitenantUsers int
+
+func TestMultitenancyLoad(t TestingTB, ctx *TestingContext) {
```

Comment:
Reading over this test I don't see what you are trying to assert as true. The only way there is a failure in the test is with the set up. Should this test not be asserting on the length of time the it takes for a user to log into 3scale?

Reply:
When test fails the message that's outputted to a user is:
`"User test-userXYZ login and creation of tenant failed with: timeout waiting for condition"`